imports/ui/explorer-components/no-item-msg.jsx

```diff
@@ -21,7 +22,15 @@ export function NoItemMsg ({item}) {
         <div className='mt3 ph4'>
           <div className='mid-gray b lh-copy'>
             <div>There are no {item}s that match your current filter combination.</div>
-            <div>Click on the "+" button to select a unit to add a new {item}</div>
+            {item === 'unit' ? (
```

Comment:
This component shouldn't be unit specific. It should be enhanced to facilitate what the unit select needs without mentioning 'unit' specifically